apps/mobile/src/components/balance/balance.tsx

```diff
@@ -19,7 +19,7 @@ export function formatBalance(balance: Money, isFiat: boolean) {
     return i18nFormatCurrency(balance, isLargeBalance ? 0 : balance.decimals);
   }
 
-  return formatMoney(balance);
+  return formatMoneyWithoutSymbol(balance);
```

Comment:
Changing this as otherwise `Runes` break the whole UI. 

![Screenshot 2025-03-07 at 15 55 29](https://github.com/user-attachments/assets/d4f5a4bd-5ca6-4872-b031-2e08b8212c91)